advanced/Scripts/update.sh

```diff
@@ -79,11 +73,15 @@ if [ ! -d "/var/www/html/admin" ]; then #This is unlikely
 fi
 
 echo "::: Checking for updates..."
-piholeVersion=$(pihole -v -p -c)
-piholeVersionLatest=$(pihole -v -p -l)
+# Checks Pi-hole version > pihole only > current local git repo version : returns string in format vX.X.X
+piholeVersion=$(/usr/local/bin/pihole -v -p -c)
```

Comment:
Pi-hole -v actually calls a subscript and passes -p and -c as arguments to that, or I'd prefer that to be `pihole --version --pihole --current` just to spell it all out.